### PR TITLE
fix/agenda-settings: Add confirm dialog on closing agenda settings drawer

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting/AgendaUserSettingDrawer.vue [hunk 1]
```diff
@@ -139,6 +150,9 @@ export default {
         return;
       }
       return true;
+    },
+    ConfirmCancelSavingChanges() {
```
Comment (boubaker) [inline agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting/AgendaUserSettingDrawer.vue]:
```suggestion
    confirmCancelSavingChanges() {
```

File: agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting/AgendaUserSettingDrawer.vue [hunk 2]
```diff
@@ -48,6 +48,13 @@
           </div>
         </v-layout>
       </v-form>
+      <exo-confirm-dialog
+        ref="CancelSavingChangesDialog"
+        :message="$t('agenda.settings.drawer.confirmCancelChanges')"
+        :title="$t('agenda.settings.drawer.confirmCancelChanges.title')"
+        :ok-label="$t('agenda.button.confirm')"
+        :cancel-label="$t('agenda.button.cancel')"
+        @ok="ConfirmCancelSavingChanges()" />
```
Comment (boubaker) [inline agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting/AgendaUserSettingDrawer.vue]:
```suggestion
        @ok="confirmCancelSavingChanges()" />
```

Comment (aelbenney) [reply agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting/AgendaUserSettingDrawer.vue]: fixed

File: agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting/AgendaUserSettingDrawer.vue (hunk 1 above)
Comment (aelbenney) [reply agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting/AgendaUserSettingDrawer.vue]: fixed

Comment (aelbenney) [reply agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting/AgendaUserSettingDrawer.vue]: fixed

**Review by boubaker — APPROVED** [review]

**Pull request merged**